cura/Settings/ExtruderManager.py

```diff
@@ -25,6 +29,7 @@ def __init__(self, parent = None):
         self._extruder_trains = { } #Per machine, a dictionary of extruder container stack IDs.
         self._active_extruder_index = 0
         UM.Application.getInstance().globalContainerStackChanged.connect(self.__globalContainerStackChanged)
+        self._global_container_stack_definition = None
```

Comment:
Since this is used to store the ID of the definition and not the actual DefinitionContainer using "_global_container_stack_definition_id" is clearer.